diff --git a/api.js b/api.js
index 44ccfaa28b23166c174db6aa8f82c204d5983518..996d0eac5e03c116dff8b9da961cfc821ab89bea 100644
--- a/api.js
+++ b/api.js
@@ -40950,10 +40950,12 @@ AND
         progressCallback("checks", checksCount, "fetching");
       }
       for (const checkConstraintRow of checkConstraints) {
-        const constraintName = checkConstraintRow["CONSTRAINT_NAME"];
-        const constraintValue = checkConstraintRow["CHECK_CLAUSE"];
-        const tableName = checkConstraintRow["TABLE_NAME"];
+        const constraintName = checkConstraintRow["CONSTRAINT_NAME"] ?? checkConstraintRow["constraint_name"];
+        const constraintValue = checkConstraintRow["CHECK_CLAUSE"] ?? checkConstraintRow["check_clause"];
+        const tableName = checkConstraintRow["TABLE_NAME"] ?? checkConstraintRow["table_name"];
         const tableInResult = result[tableName];
+        if (typeof tableInResult === "undefined")
+          continue;
         tableInResult.checkConstraint[constraintName] = {
           name: constraintName,
           value: constraintValue
diff --git a/api.mjs b/api.mjs
index d52ed68b1a8da7e334b1d2e08b4298cd3a9597df..040899ad3d66977adfd54e397d13b11e827aede7 100644
--- a/api.mjs
+++ b/api.mjs
@@ -40955,10 +40955,12 @@ AND
         progressCallback("checks", checksCount, "fetching");
       }
       for (const checkConstraintRow of checkConstraints) {
-        const constraintName = checkConstraintRow["CONSTRAINT_NAME"];
-        const constraintValue = checkConstraintRow["CHECK_CLAUSE"];
-        const tableName = checkConstraintRow["TABLE_NAME"];
+        const constraintName = checkConstraintRow["CONSTRAINT_NAME"] ?? checkConstraintRow["constraint_name"];
+        const constraintValue = checkConstraintRow["CHECK_CLAUSE"] ?? checkConstraintRow["check_clause"];
+        const tableName = checkConstraintRow["TABLE_NAME"] ?? checkConstraintRow["table_name"];
         const tableInResult = result[tableName];
+        if (typeof tableInResult === "undefined")
+          continue;
         tableInResult.checkConstraint[constraintName] = {
           name: constraintName,
           value: constraintValue
diff --git a/bin.cjs b/bin.cjs
index 70941f2a5aec6dcd1218e8a699fb5aafb4025f35..7529c463d49a3764b513ae38f96176acb8a5a6ab 100755
--- a/bin.cjs
+++ b/bin.cjs
@@ -18240,10 +18240,12 @@ AND
         progressCallback("checks", checksCount, "fetching");
       }
       for (const checkConstraintRow of checkConstraints) {
-        const constraintName = checkConstraintRow["CONSTRAINT_NAME"];
-        const constraintValue = checkConstraintRow["CHECK_CLAUSE"];
-        const tableName = checkConstraintRow["TABLE_NAME"];
+        const constraintName = checkConstraintRow["CONSTRAINT_NAME"] ?? checkConstraintRow["constraint_name"];
+        const constraintValue = checkConstraintRow["CHECK_CLAUSE"] ?? checkConstraintRow["check_clause"];
+        const tableName = checkConstraintRow["TABLE_NAME"] ?? checkConstraintRow["table_name"];
         const tableInResult = result[tableName];
+        if (typeof tableInResult === "undefined")
+          continue;
         tableInResult.checkConstraint[constraintName] = {
           name: constraintName,
           value: constraintValue
